docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>Similarly here I would call <code>scipy.stats.normal.logpdf</code> and sum the output, as that is exactly what this PyMC example is doing.</p><p> </p><p>Or you don't need to sum here and below neither</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
<p>+1</p>

Reply:
+1